fog/api/proto/view.proto

```diff
@@ -58,9 +52,15 @@ message MultiViewStoreQueryResponse {
     //  query.
     attest.Message query_response = 1;
 
+    /// The FogViewStoreUri for the specific Fog View Store that
+    /// tried to decrypt the MultiViewStoreQueryRequest and failed.
+    /// The client should subsequently authenticate with the machine
+    /// described by this URI.
```

Comment:
might consider re-wording this comment since it's now part of `MultiViewStoreQueryResponse` and used elsewhere besides handling of `FogViewStoreDecryptionError`.
```suggestion
    /// The FogViewStoreUri for the responding Fog View Store.
```

Reply:
Done.